contributions/echonetExampleTD.td.jsonld

```diff
@@ -0,0 +1,114 @@
+{
+	"id": "urn:simple",
+	"@context": "https://www.w3.org/2019/wot/td/v1",
+    "eoj": "0x0290",
+	"title": "General Lighting",
+	"descriptions": {"ja": "一般照明","en": "General Lighting"},
+	"securityDefinitions": {
+		"basic_sc": {
+			"scheme": "basic",
+			"in": "header"
+		}
+	},
+	"security": [
+		"basic_sc"
+	],
+
+	"properties": {
+		"brightness": {
+            "type":"object",
+            "properties":{
+                "brightness": {
+			        "type": "number"
+                }
+            },
+            "epc": "0xB0",
+            "descriptions": { "ja": "照度レベル設定","en": "Illuminance level"},
+            "observable":false,
+			"forms": [
+				{
+					"href": "/elapi/v1/devices/<deviceId>/properties/brightness",
```

Comment:
`<deviceId>` is not a URI variable, right? At first glace it might confuse people.

Reply:
It is one as far as I understood. This is a bit of a TM than a TD since it does not have the concrete endpoints. I would imagine that there will be something like `abc123` replacing the `<deviceId>`

Reply:
Shouldn't it than be something like

`"/elapi/v1/devices/{{deviceId}}/properties/brightness"`

Reply:
If we are talking about a TM yes, here I just copied their URLs of the slide. Ideally there should be the concrete URI here.